apps/server/src/modules/learnroom/controller/dashboard.controller.ts

```diff
@@ -20,9 +19,9 @@ export class DashboardController {
 		return dto;
 	}
 
-	@Patch(':id/moveElement')
+	@Patch(':dashboardId/moveElement')
 	async moveElement(
-		@Param('id', ParseObjectIdPipe) dashboardId: string,
+		@Param() { dashboardId }: DashboardUrlParams,
```

Comment:
```suggestion
		@Param() params: DashboardUrlParams,
```

Just one question: is this more readable than that?
And then continue to work with

`params.dashboardId`